app/_src/gateway/get-started/ai-gateway.md

```diff
@@ -1,93 +1,287 @@
+  -d '{
+     "model": "gpt-4o-mini",
+     "messages": [{"role": "user", "content": "Say this is a test!"}]
+   }'
+```
+
+You should get a response that looks like this, containing the response `This is a test!`:
+
+```json
+{
+  "id": "chatcmpl-AIm1TMhTkcH1sf67GYXIM5fsfu94X9Gdk",
+  "object": "chat.completion",
+  "created": 1729037867,
+  "model": "gpt-4o-mini-2024-07-18",
+  "choices": [
+    {
+      "index": 0,
+      "message": {
+        "role": "assistant",
+        "content": "This is a test! How can I assist you today?",
+        "refusal": null
+      },
+      "logprobs": null,
+      "finish_reason": "stop"
+    }
+  ],
+  "usage": {
+    "prompt_tokens": 13,
+    "completion_tokens": 12,
+    "total_tokens": 25,
+    "prompt_tokens_details": {
+      "cached_tokens": 0
+    },
+    "completion_tokens_details": {
+      "reasoning_tokens": 0
+    }
+  },
+  "system_fingerprint": "fp_r0bdr52e6e"
+}
+```
+
+And just like that, your traffic is now being properly proxied to OpenAI via {{site.base_gateway}} and its AI plugins!
+
+## Installing more AI plugins
+
+The AI Proxy and AI Proxy Advanced plugins are able to understand the incoming OpenAI protocol, 
+so you can now further apply deep capabilities like:
+
+* Route to [all other LLMs](/hub/kong-inc/ai-proxy-advanced/) that we support, 
+even the ones that don't natively support the OpenAI specification, as Kong will automatically transform the request. 
+Write once, and use all LLMs.
+* Extract [observability metrics for AI](/gateway/latest/production/monitoring/ai-metrics/).
+* Accelerate traffic with the [AI Semantic cache plugin](/hub/kong-inc/ai-semantic-cache/) plugin.
+* Secure traffic with the [AI Prompt Guard](/hub/kong-inc/ai-prompt-guard/) and 
+[AI Semantic Prompt Guard](/hub/kong-inc/ai-semantic-prompt-guard/) plugins.
+* Provide prompt templates with [AI Prompt template](/hub/kong-inc/ai-prompt-template/).
+* Automatically inject system or assistant prompts on every incoming prompt via the 
+[AI Prompt Decorator](/hub/kong-inc/ai-prompt-decorator/).
+
+See all the [AI plugins](/hub/?category=ai) for more capabilities.
+
+For example, you can rate limit AI traffic based on the number of tokens that are being sent 
+(as opposed to the number of API requests) using the [AI Rate Limiting Advanced](/hub/kong-inc/ai-rate-limiting-advanced/) plugin:
+
+```sh
+curl -X POST http://localhost:8001/services/llm_service/plugins \
+    --header "accept: application/json" \
+    --header "Content-Type: application/json" \
+    --data '
+    {
+  "name": "ai-rate-limiting-advanced",
+  "config": {
+    "llm_providers": [
+      {
+        "name": "openai",
+        "limit": 5,
+        "window_size": 60
+      }
+    ]
+  }
+}'
+```
+
+Every other {{site.base_gateway}} plugin can also be used in addition to the AI plugins, for advanced access control, authorization and authentication, security, observability, and more.
+
+## Quickstart script
```

Comment:
I think we can delete this from here and throw it in: 
https://deploy-preview-8056--kongdocs.netlify.app/gateway/latest/ai-gateway/

But I don't feel very strongly about it so feel free to keep it.

Reply:
I'm good with that, and I think it would be a better compromise for this whole update.